File: lint/src/test/java/net/grandcentrix/thirtyinch/IssueRegistryTest.java
package net.grandcentrix.thirtyinch;

import org.junit.Test;

import static org.fest.assertions.api.Assertions.assertThat;

public class IssueRegistryTest {

    @Test public void testIssueList() {
        assertThat(new IssueRegistry().getIssues()).containsExactly(
                MissingTiViewImplementationDetector.ISSUE
        );
    }
}

File: lint/src/test/kotlin/net/grandcentrix/thirtyinch/IssueRegistryTest.kt
package net.grandcentrix.thirtyinch

import com.android.tools.lint.detector.api.Issue
import org.fest.assertions.api.Assertions.assertThat
import org.junit.Test

class IssueRegistryTest {

    @Test fun testIssueList() {
        assertThat<Issue>(IssueRegistry().issues)
                .containsExactly(
                        MissingViewInThirtyInchDetector.ISSUE,
                        MissingViewInCompositeDetector.ISSUE
                )
    }
}
